var/spack/repos/builtin/packages/py-yapf/package.py

```diff
@@ -13,5 +13,6 @@ class PyYapf(PythonPackage):
     url      = "https://github.com/google/yapf/archive/v0.2.1.tar.gz"
 
     version('0.2.1', sha256='13158055acd8e3c2f3a577528051a1c5057237f699150211a86fb405c4ea3936')
+    version('0.30.0', sha256='9f561af26f8d27c3a334d3d2ee8947b8826a86691087e447ce483512d834682c')
```

Comment:
Can you sort these from newest to oldest?

Reply:
Done in [54a99cb](https://github.com/spack/spack/pull/19598/commits/54a99cbc43bd984f457853104dc4e6cd5d3d4877).